runtime/tests/port/si.c

```diff
@@ -1892,10 +1892,22 @@ j9sysinfo_test_get_cwd3(struct J9PortLibrary *portLibrary)
 	} else {
 		outputComment(PORTLIB, "CWD = %s\n", buffer);
 	}
-
-	rc = memcmp(utf8, buffer, strlen(buffer));
-	if (0 != rc) {
-		outputErrorMessage(PORTTEST_ERROR_ARGS, "invalid directory rc: %d\n", rc);
+	
+	
+	{
+		char *cmpBuffer = buffer;
+#if defined(J9ZOS390)
+		/* On z/OS, the /tmp directory is typically found in /SYSTEM. */
```

Comment:
OSX appears to have a similar issue with /tmp being /private/tmp handled above by:
```
#if defined(OSX)
	const char *utf8 = "/private/tmp/j9sysinfo_test_get_cwd3/";
#else /* defined(OSX) */
```
Can this be adapted for Z to always use `/SYSTEM/tmp/j9sysinfo_test_get_cwd3/`?

Reply:
It doesn't work on the older machine, but I can do what Keith is suggesting.